sdk/entra/WebJobs.Extensions.AuthenticationEvents/test/OpenApi/TokenIssuanceStartPreview10012021.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using payloads = WebJobs.Extensions.AuthenticationEvents.Tests.Payloads;
+namespace WebJobs.Extensions.AuthenticationEvents.Tests.OpenApi
+{
+    /// <summary>Class to house open api tests for the OnTokenIssuanceStart event version preview_10_01_2021</summary>
+    public class OpenApi
```

Comment:
The file name should match the class name;  we do not generally recommend tying the name to a specific API version.    Can we drive the API version from data?